build/build-image/cache/cloudbuild.yaml

```diff
@@ -137,6 +137,9 @@ steps:
       - name: original
         path: /original
 
+substitutions:
```

Comment:
ohh right! I was initially adding this substitution in the target, but we later decided not to use any target